auth-api/src/auth_api/services/products.py

```diff
@@ -85,6 +88,18 @@ def create_product_subscription(org_id, subscription_data: Dict[str, Any], is_ne
                                                                 product_code=product_code,
                                                                 status_code=product_model.default_subscription_status)\
                     .flush()
+                # create a staff review task for this product subscription
+                user = UserModel.find_by_jwt_token(token=token_info)
+                task_info = {'name': product_model.description,
+                             'relationshipId': product_subscription.id,
+                             'relatedTo': user.id,
+                             'dateSubmitted': datetime.today(),
+                             'relationshipType': TaskRelationshipType.PRODUCT.value,
+                             'type': TaskType.PENDING_STAFF_REVIEW.value,
```

Comment:
I think we have a way to find the status ; something like default status of a product;

Reply:
As per our discussion. need to discuss more shall do at a later PR